app/components/collections/edit/pane_component.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= render Edit::TabForm::PaneComponent.new(**pane_args) do |component| %>
```

Comment:
Works and collections are getting their own pane components because they each need different buttons.